simulation_ws/src/test_nodes/launch/docking_test.launch

```diff
@@ -0,0 +1,31 @@
+<launch>
```

Comment:
By convention, environment variables go in `UPPER_CASE`, while the param names are in `snake_case` like Python.
I'll add an example below